crates/astria-sequencer/src/bridge/storage/keys.rs

```diff
@@ -96,9 +91,6 @@ mod tests {
 
     #[test]
     fn keys_should_not_change() {
-        insta::assert_snapshot!(INIT_BRIDGE_ACCOUNT_BASE_FEE);
```

Comment:
It's hard to track what exactly changed because all the tests get renumbered.

Shouldn't do it in this PR, but give unique names to all the snapshot asserts in a future PR.

Reply:
Tracking issue: https://github.com/astriaorg/astria/issues/1656

Reply:
Related..... I don't believe this is removing the snapshots which are no longer used. Because the numbers changed and we just have fewer.

Reply:
I deleted and regenerated all the snapshots in `fees/storage/snapshots`, so hopefully they should all be up to date?